### PR TITLE
Navigate to landing page after hangup.

### PR DESCRIPTION
**Pull request by chuckhays**
This is all it would take to redirect back to the landing page after hitting the hang up button.

@juberti

### CODE REVIEW

File: samples/web/content/apprtc/js/appcontroller.js [hunk 1]
```diff
@@ -179,6 +179,9 @@ AppController.prototype.hangup_ = function() {
   this.transitionToDone_();
 
   this.call_.hangup();
+
+  trace('Reloading main page.');
```
Comment (juberti) [inline samples/web/content/apprtc/js/appcontroller.js]: I tried this, but I think we will want to retain the end-of-call page, perhaps to display a call rating widget or some such. But let's add a link in the status area to take you back to the landing page, e.g. [Click here] to rejoin, or [start a new room].

Also, can we make the rejoin-room link immediately put you back into the room (i.e. act as if you clicked Join from the landing page)?

Comment (chuckhays) [inline samples/web/content/apprtc/js/appcontroller.js]: Yeah, that makes sense. I'll take a look.

**Comment by chuckhays** [comment]
Added a new room link like you suggested. Not sure I like the text as it is.

Maybe it should be something like:
You have left the call: [rejoin] or start a [new room].

screen shot 2015-01-26 at 8 26 25 am

**Comment by chuckhays** [comment]
> Also, can we make the rejoin-room link immediately put you back into the room (i.e. act as if you clicked Join from the landing page)?

In what way? Removing the page navigation?

**Comment by samdutton** [comment]
LGTM.

**Comment by jiayliu** [comment]
This won't work for the Chrome App, right?

**Comment by juberti** [comment]
New text SGTM.

Re: rejoin action, I think the navigation might be fine, as long as it works in the Chrome app.

**Comment by chuckhays** [comment]
It won't work as is for the Chrome app, needs to have a click handler hooked up. I'll have to check how the code responds to making another call without a "reload". We'll have to take on that work eventually, I was just avoiding it to work on our other priorities.

**Comment by chuckhays** [comment]
This change should work with both the Chrome App and the web app. There are fixes needed in the Chrome App, as the hang-up functionality doesn't work properly due to using sync xhr, which isn't allowed. This could be fixed at the same time as issue #437.

screen shot 2015-01-27 at 11 43 02 am

**Comment by juberti** [comment]
lgtm

**Pull request merged**